ldms/src/ldmsd/ldmsd_request.c

```diff
@@ -6348,33 +6311,24 @@ static int stream_client_dump_handler(ldmsd_req_ctxt_t reqc)
 
 void stream_xprt_term(ldms_t x)
 {
-	struct sockaddr local_sa;
 	__RSE_t ent;
 	struct rbn *rbn;
 	char _buff[sizeof(struct __RSE_key_s) + 256] = {};
```

Comment:
The _RSE_key_s is not correct. The stream name is fixed at 256B, but the stream name size is arbitrary. We either need to clamp the stream name length or we need to make _name_ a pointer to a dup'd stream name string. We need this patch now, so I will accept it, but I'd like you to fix this and submit another pull request. I think the simplest thing is to make _name_ point to a dup'd string and handle the free of _name_ in the _RSE_key_s cleanup  logic.